qemu/tests/netkvm_protocol_binding.py

```diff
@@ -58,9 +66,19 @@ def run(test, params, env):
 
     test.log.info("Will install inf file found at '%s'", inf_path)
     install_cmd = params["install_cmd"] % inf_path
-    status, output = session.cmd_status_output(install_cmd, timeout=timeout)
-    if status:
-        test.error("Install inf file failed, output=%s" % output)
+
+    # workaround for driver no signature
+    key_to_install_driver = params.get("key_to_install_driver").split(";")
+    thread_installation = utils_misc.InterruptedThread(
+        session.cmd_status_output, (install_cmd, timeout))
+    try:
+        thread_installation.start()
+        time.sleep(10)
+        send_key(vm, key_to_install_driver)
+        thread_installation.join()
+    except Exception as err:
+        err_msg = "Error Info: '%s'"
+        test.error(err_msg % err)
```

Comment:
```suggestion
    try:
        session.cmd_status_output(install_cmd, timeout)
    except aexpect.ShellTimeoutError:
        send_key(vm, key_to_install_driver)
        # Add some checking if the driver is successfully installed here
```

Reply:
I don't think we need to wait for it to timeout, we can just send the key directly to the guest to ensure a successful installation.
If we wait for the install to timeout, the warning popup will disappear, so we can't send the key to install the driver.